api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -362,14 +362,30 @@ message HeaderMutation {
   repeated string remove_headers = 2;
 }
 
-// Replace the entire message body chunk received in the corresponding
-// HttpBody message with this new body, or clear the body.
+// The body response message corresponding to FULL_DUPLEX_STREAMED body mode.
+message StreamedBodyResponse {
+  // Envoy sends this body mutation out as it is.
```

Comment:
Should this be "The body response segment/chunk that will be passed to the upstream/downstream."?

Reply:
done